.github/workflows/checks.yml

```diff
@@ -47,7 +47,7 @@ jobs:
     - name: Check imports alphabetized
       run: pipenv run isort --check-only ./app ./tests
     - name: Run tests with coverage
-      run: pipenv run coverage run --omit=*/notifications_utils/* -m pytest -n4 --maxfail=10
+      run: pipenv run coverage run --omit=*/notifications_utils/* -m pytest --maxfail=10
```

Comment:
Ah, this is what would eliminate the concurrent worker processes for tests?

@stvnrlly - if you're okay with doing this too, this would likely unblock me on the dependency update PR, and we could revisit this piece separately at a future time to re-enable the concurrent processes once we figure out what's up with the test configuration that is blowing things up with concurrent workers.

Reply:
Yep, that's fine by me.